src/endianconv.h

```diff
@@ -34,6 +34,7 @@
 #define __ENDIANCONV_H
 
 #include "config.h"
+
```

Comment:
@filipecosta90 notice that this files comes out of redis this is also the reason why we don't include it in the shared object.

Reply:
fixed in the last commit